[FritzBox TR-064] Can´t get WAN IP address

Hi!

There´s an problem with the FritzBox TR-064 binding that results in some errors when trying to get the current WAN IP address.

The openHAB log says:
`2017-08-22 15:59:23.920 [ERROR] [o.o.b.f.i.FritzboxTr064Binding] - FritzBox returned unexpected response. Could not find expected datavalue NewExternalIPAddress in response <?xml version="1.0"?><s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/" s:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/">
<s:Body>
<s:Fault>
<faultcode>s:Client</faultcode>
<faultstring>UPnPError</faultstring>
<detail>
<UPnPError xmlns="urn:dslforum-org:control-1-0">
<errorCode>401</errorCode>
<errorDescription>Invalid Action</errorDescription>
</UPnPError>
</detail>
</s:Fault>
</s:Body>
</s:Envelope>`

It seems like the binding is trying to get the value at an now unknown place.

## Possible Solution
All informations for TR-064 can be found at AVM.
https://avm.de/fileadmin/user_upload/Global/Service/Schnittstellen/wancommonifconfigSCPD.pdf

## Steps to Reproduce (for bugs)
1. Add the org.openhab.binding.fritzboxtr064-1.9.0-SNAPSHOT.jar binding
2. Configure the openhab.cfg
3. Add the item to default.items
`String fbWAN "IP-Adresse: [%s]" <fritzbox> {fritzboxtr064="wanip"}`
4. Add the item to the sitemap
`Text item=fbWAN`
5. Get the error in the openhab.log

## Context
I´m not able to get my current IP address and send messages when my current IP address changes.

## Your Environment
* Version used: openHAB 1.8.3 / FritzBox TR-064 binding 1.9.0 SNAPSHOT

Just checked the DEBUG loglevel and there´s no additional information. Just the same error as before:
`2017-08-28 15:24:43.460 [ERROR] [o.o.b.f.i.FritzboxTr064Binding] - FritzBox returned unexpected response. Could not find expected datavalue NewExternalIPAddress in response <?xml version="1.0"?><s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/" s:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/">
<s:Body>
<s:Fault>
<faultcode>s:Client</faultcode>
<faultstring>UPnPError</faultstring>
<detail>
<UPnPError xmlns="urn:dslforum-org:control-1-0">
<errorCode>401</errorCode>
<errorDescription>Invalid Action</errorDescription>
</UPnPError>
</detail>
</s:Fault>
</s:Body>
</s:Envelope>`

Seems like the logging capabilites of the FritzBoxTR064 binding aren´t enough...

Same error:
> 2017-08-30 17:40:56.340 [DEBUG] [o.o.b.f.i.FritzboxTr064Binding] - Raw SOAP Request to be sent to FritzBox: <SOAP-ENV:Envelope xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/"><SOAP-ENV:Header/><SOAP-ENV:Body><u:GetExternalIPAddress xmlns:u="urn:dslforum-org:service:WANPPPConnection:1"/></SOAP-ENV:Body></SOAP-ENV:Envelope>
> 2017-08-30 17:40:56.418 [DEBUG] [o.o.b.f.i.FritzboxTr064Binding] - Releasing connection
> 2017-08-30 17:40:56.419 [DEBUG] [o.o.b.f.i.FritzboxTr064Binding] - Raw SOAP Response from FritzBox: <?xml version="1.0"?>
> <s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/" s:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/">
> <s:Body>
> <s:Fault>
> <faultcode>s:Client</faultcode>
> <faultstring>UPnPError</faultstring>
> <detail>
> <UPnPError xmlns="urn:dslforum-org:control-1-0">
> <errorCode>401</errorCode>
> <errorDescription>Invalid Action</errorDescription>
> </UPnPError>
> </detail>
> </s:Fault>
> </s:Body>
> </s:Envelope>
> 2017-08-30 17:40:56.439 [ERROR] [o.o.b.f.i.FritzboxTr064Binding] - FritzBox returned unexpected response. Could not find expected datavalue NewExternalIPAddress in response <?xml version="1.0"?><s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/" s:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/">
> <s:Body>
> <s:Fault>
> <faultcode>s:Client</faultcode>
> <faultstring>UPnPError</faultstring>
> <detail>
> <UPnPError xmlns="urn:dslforum-org:control-1-0">
> <errorCode>401</errorCode>
> <errorDescription>Invalid Action</errorDescription>
> </UPnPError>
> </detail>
> </s:Fault>
> </s:Body>
> </s:Envelope>

--- a/bundles/binding/org.openhab.binding.fritzboxtr064/README.md
+++ b/bundles/binding/org.openhab.binding.fritzboxtr064/README.md
@@ -45,7 +45,10 @@ This binding can be configured in the file `services/fritzboxtr064.cfg`.
 
 ```
 String  fboxName            "FBox Model [%s]"           {fritzboxtr064="modelName"}
+# get wan ip if FritzBox establishes the internet connection (e. g. via DSL)
 String  fboxWanIP           "FBox WAN IP [%s]"          {fritzboxtr064="wanip"}
+# get wan ip if FritzBox uses internet connection of external router
+String  fboxWanIPExternal   "FBox external WAN IP [%s]" {fritzboxtr064="externalWanip"}
 Switch  fboxWifi24          "2,4GHz Wifi"               {fritzboxtr064="wifi24Switch"}
 Switch  fboxWifi50          "5,0GHz Wifi"               {fritzboxtr064="wifi50Switch"}
 Switch  fboxGuestWifi       "Guest Wifi"                {fritzboxtr064="wifiGuestSwitch"}

--- a/bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/Tr064Comm.java
+++ b/bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/Tr064Comm.java
@@ -676,6 +676,8 @@ public class Tr064Comm {
         _alItemMap.add(new ItemMap("modelName", "GetInfo", "DeviceInfo-com:serviceId:DeviceInfo1", "", "NewModelName"));
         _alItemMap.add(new ItemMap("wanip", "GetExternalIPAddress",
                 "urn:WANPPPConnection-com:serviceId:WANPPPConnection1", "", "NewExternalIPAddress"));
+        _alItemMap.add(new ItemMap("externalWanip", "GetExternalIPAddress",
+                "urn:WANIPConnection-com:serviceId:WANIPConnection1", "", "NewExternalIPAddress"));
 
         // Wifi 2,4GHz
         ItemMap imWifi24Switch = new ItemMap("wifi24Switch", "GetInfo",

